llvm/test/Transforms/LoopVectorize/X86/interleaved-accesses-use-after-free.ll

```diff
@@ -24,27 +24,10 @@ target triple = "x86_64-redhat-linux-gnu"
 %struct.baz = type { i64, %struct.pluto }
 
 ; LOOP-ACCESS: Too many dependences, stopped recording
-
-; If no dependences are recorded because there are too many, LoopAccessAnalysis
-; just conservatively returns true for any pair of instructions compared (even
-; those belonging to the same store group). This tests make sure that we do not
-; incorrectly release a store group which had no dependences between its
-; members, even if we have no dependences recorded because there are too many. 
-
-; CHECK: LV: Creating an interleave group with:  store ptr null, ptr %phi5, align 8
-; CHECK: LV: Inserted:  store ptr %load12, ptr %getelementptr11, align 8
-; CHECK:     into the interleave group with  store ptr null, ptr %phi5
-; CHECK: LV: Inserted:  store ptr %load7, ptr %getelementptr, align 8
-; CHECK:     into the interleave group with  store ptr null, ptr %phi5
-
-; CHECK: LV: Creating an interleave group with:  store ptr null, ptr %getelementptr13, align 8
-; CHECK: LV: Inserted:  store ptr null, ptr %phi6, align 8
-; CHECK:     into the interleave group with  store ptr null, ptr %getelementptr13
-; CHECK: LV: Invalidated store group due to dependence between   store ptr %load7, ptr %getelementptr, align 8 and   store ptr null, ptr %getelementptr13, align 8
-; CHECK-NOT: LV: Invalidated store group due to dependence between
+; LV: Can't vectorize due to memory conflicts
```

Comment:
Is this also a regression?

Reply:
Sorry, made a mistake when resolving conflicts: the unrelated SCEVCouldNotCompute distance is responsible for this.